backend/src/main/java/com/webappsbusters/parcelmanagement/domain/ParcelDto.java

```diff
@@ -22,22 +25,24 @@
     private int width;
     private ParcelSizeDto size;
     private Duration timeToDeliver;
+    private String name;
 
     @Builder
     public static class ParcelDtoBuilder {
-        private String parcelId;
-        private ParcelStatusDto status;
-        private String senderCity;
-        private String senderPostCode;
-        private String senderStreet;
-        private String receiverCity;
-        private String receiverPostCode;
-        private String receiverStreet;
-        private int weightInKg;
-        private int height;
-        private int length;
-        private int width;
-        private ParcelSizeDto size;
-        private Duration timeToDeliver;
+        private final String parcelId;
+        private final ParcelStatusDto status;
+        private final String senderCity;
+        private final String senderPostCode;
+        private final String senderStreet;
+        private final String receiverCity;
```

Comment:
wont there be a problem later on when we want to change receiver adress (PAC-20) while variables are made final?

Reply:
No, as this is a builder, variables of parcel are not final.